internal/web/templates/components/pkg-detail-header.html

```diff
@@ -114,7 +114,26 @@ <h1 class="text-reset m-0">
             Auto-Update:
             <strong>{{ .AutoUpdate }}</strong>
           </span>
+          <span
+            class="badge p-1 fw-normal {{ if eq .Status.Status " Failed" }}
+              bg-danger text-white border border-1 border-danger
+            {{ else }}
+              bg-body-secondary text-primary-emphasis border-primary border border-1 p-1 fw-normal
+            {{ end }}"
+            data-bs-toggle="popover"
```

Comment:
I think all these `data-bs-` attributes can be removed now that we don't want any tooltip/popover, right?